src/bin.ts

```diff
@@ -41,7 +42,7 @@ const unicodeConfig = process.env.TYPINGS_CONFIG_UNICODE || process.env.NPM_CONF
 
 const argv = minimist<Argv>(process.argv.slice(2), {
   boolean: ['version', 'save', 'saveDev', 'savePeer', 'global', 'verbose', 'production', 'unicode'],
-  string: ['cwd', 'out', 'name', 'source', 'offset', 'limit', 'sort'],
+  string: ['cwd', 'out', 'name', 'source', 'offset', 'limit', 'sort', 'logLevel'],
```

Comment:
Can you consistently rename `logLevel` to `loglevel`. It's typically all lowercase (https://docs.npmjs.com/misc/config).